File: trail1/범위 안의 수 2/number-in-range-2.c
#include <stdio.h>

int main() {
    int cnt = 0, ssum = 0;
    for(int i=0; i<10; i++) {
        int a;
        scanf("%d", &a);
        if (a>=0 && a<=200) {cnt++; ssum += a;}
    }
    printf("%d %.1lf", ssum, (double)ssum/cnt);
    return 0;
}

File: trail1/범위 안의 수 2/number-in-range-2.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int ssum = 0, cnt = 0;
        for (int i=0; i<10; i++) {
            int a = int.Parse(Console.ReadLine());
            if (a >= 0 && a <= 200) {
                cnt += 1; 
                ssum += a;
            }
        }
        Console.WriteLine($"{ssum} {(double)ssum/cnt:F1}");
    }
}
